Client/Application/AppDelegate.swift

```diff
@@ -144,6 +144,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         self.updateAuthenticationInfo()
         SystemUtils.onFirstRun()
 
+        RustFirefoxAccounts.startup(prefs: profile.prefs).uponQueue(.main) { _ in
+            print("RustFirefoxAccounts started")
+        }
```

Comment:
This change is moving the startup to earlier in the launch. Should be no different but I can remove the 'if let = profile' bit that was used in the old code.